OpenRA.Mods.Common/Traits/C4Demolition.cs

```diff
@@ -23,18 +23,22 @@ class C4DemolitionInfo : ITraitInfo
 			"Measured in game ticks. Default is 1.8 seconds.")]
 		public readonly int C4Delay = 45;
 
-		[Desc("Number of times to flash the target")]
+		[Desc("Number of times to flash the target.")]
 		public readonly int Flashes = 3;
 
-		[Desc("Delay before the flashing starts")]
+		[Desc("Delay before the flashing starts.")]
 		public readonly int FlashesDelay = 4;
 
-		[Desc("Interval between each flash")]
+		[Desc("Interval between each flash.")]
 		public readonly int FlashInterval = 4;
 
-		[Desc("Duration of each flash")]
+		[Desc("Duration of each flash.")]
 		public readonly int FlashDuration = 3;
 
+		[Desc("Behaviour when entering the structure.",
+			"Possible values are Temporary, Suicide, Dispose.")]
+		public readonly EnterBehaviour EnterBehaviour = EnterBehaviour.Temporary;
```

Comment:
"Temporary" is hard to understand for me without any explanation. How about "Escape", "Evacuate", "Exit", "Leave", "Withdraw" or other synonyms?